<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>403</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="navLayout">
     <item>
      <widget class="QLabel" name="label_Sort">
       <property name="text">
        <string>排序方法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_Sort">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Sort_info">
       <property name="text">
        <string>排序指标</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_Sort_info">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Find_method">
       <property name="text">
        <string>查找方法</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_Find">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_Find_info">
       <property name="text">
        <string>查找字段</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cbox_Find_txt">
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txt_sort"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTableWidget" name="tabWidget"/>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>当前：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="currentPageLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>总页数：</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="countPageLabel">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="upPushButton">
       <property name="text">
        <string>上一页</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="downPushButton">
       <property name="text">
        <string>下一页</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
